src/main/scala/wdltool/GraphPrint.scala

```diff
@@ -10,15 +10,12 @@ import scala.collection.JavaConverters._
 
 import cats._
 import cats.implicits._
+import cats.derived.monoid._, legacy._
```

Comment:
Honestly I don't know, the source has 
```scala
@reexports[MkEmpty, MkMonoid, MkSemigroup]
object monoid {
  @imports[Monoid]
  object legacy
}
```
which could mean anything since it's using annotations.